Content.Shared/Medical/Cryogenics/SharedCryoPodSystem.cs

```diff
@@ -120,18 +121,49 @@ protected void AddAlternativeVerbs(EntityUid uid, SharedCryoPodComponent cryoPod
             return;
 
         // Eject verb
-        if (cryoPodComponent.BodyContainer.ContainedEntity != null)
+        if (IsOccupied(cryoPodComponent))
         {
-            args.Verbs.Add(new AlternativeVerb
-            {
-                Text = Loc.GetString("cryo-pod-verb-noun-occupant"),
-                Category = VerbCategory.Eject,
-                Priority = 1, // Promote to top to make ejecting the ALT-click action
-                Act = () => TryEjectBody(uid, args.User, cryoPodComponent)
-            });
+            AlternativeVerb verb = new();
```

Comment:
do `var` as the type, and add the type in the constructor/`new()` statement instead